app/classifier/drawing-tools/root.cjsx

```diff
@@ -62,14 +63,16 @@ module.exports = React.createClass
             true
 
         <StickyModalForm ref="detailsForm" style={SEMI_MODAL_FORM_STYLE} underlayStyle={SEMI_MODAL_UNDERLAY_STYLE} onSubmit={@handleDetailsFormClose} onCancel={@handleDetailsFormClose}>
-          {for detailTask, i in toolProps.details
-            detailTask._key ?= Math.random()
-            TaskComponent = tasks[detailTask.type]
-            <TaskComponent autoFocus={i is 0} key={detailTask._key} task={detailTask} annotation={toolProps.mark.details[i]} onChange={@handleDetailsChange.bind this, i} />}
-          <hr />
-          <p style={textAlign: 'center'}>
-            <button type="submit" className="standard-button" disabled={not detailsAreComplete}>OK</button>
-          </p>
+          <ModalFocus onEscape={@handleDetailsFormClose}>
+            {for detailTask, i in toolProps.details
+              detailTask._key ?= Math.random()
+              TaskComponent = tasks[detailTask.type]
+              <TaskComponent autoFocus={i is 0} key={detailTask._key} task={detailTask} annotation={toolProps.mark.details[i]} onChange={@handleDetailsChange.bind this, i} />}
+            <hr />
+            <p style={textAlign: 'center'}>
+              <button autoFocus type="submit" className="standard-button" disabled={not detailsAreComplete}>OK</button>
```

Comment:
I think the autoFocus on the "OK" button is overriding the autoFocus specified for the textarea in [text subTasks that are not required](https://modal-focus.pfe-preview.zooniverse.org/projects/scribe-2/project-state-testing/classify).